providers/aws/iam/policies.go

```diff
@@ -22,14 +24,49 @@ type Policy struct {
 	PolicyId                      *string `neo:"unique"`
 	PolicyName                    *string
 	UpdateDate                    *time.Time
+	PolicyVersions                []*PolicyVersion `gorm:"constraint:OnDelete:CASCADE;"`
 }
 
 func (Policy) TableName() string {
 	return "aws_iam_policies"
 }
 
-func (c *Client) transformPolicy(value *iam.Policy) *Policy {
-	return &Policy{
+type PolicyVersion struct {
+	ID               uint   `gorm:"primarykey`
+	PolicyID         uint   `neo:"ignore"`
+	AccountID        string `gorm:"-"`
+	VersionID        *string
+	CreateDate       *time.Time
+	Document         *string
+	IsDefaultVersion *bool
+}
+
+func (PolicyVersion) TableName() string {
+	return "aws_iam_policy_versions"
+}
+
+func (c *Client) transformPolicyVersionList(values []*iam.PolicyVersion) ([]*PolicyVersion, error) {
+	var tValues []*PolicyVersion
+
+	for _, value := range values {
+		decodedDocument, err := url.QueryUnescape(*value.Document)
```

Comment:
I think there should be a check here `if value.Document != nil` and initialise `decodedDocument` with with `nil` before that. Otherwise the program may panic (theoretically as I'm not sure if it possible for the document to be empty) .